internal/operators/cnv/cnv_operator.go

```diff
@@ -129,18 +138,56 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 }
 
 // GetHostRequirements provides operator's requirements towards the host
-func (o *operator) GetHostRequirements(_ context.Context, _ *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
+func (o *operator) GetHostRequirements(ctx context.Context, _ *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
+	log := logutil.FromContext(ctx, o.log)
 	switch host.Role {
 	case models.HostRoleMaster:
 		return &models.ClusterHostRequirementsDetails{
-			CPUCores: masterCPU,
-			RAMMib:   masterMemory,
+			CPUCores: MasterCPU,
+			RAMMib:   MasterMemory,
 		}, nil
 	case models.HostRoleWorker, models.HostRoleAutoAssign:
+		overhead, err := o.getDevicesMemoryOverhead(host)
+		if err != nil {
+			log.WithError(err).WithField("inventory", host.Inventory).Errorf("Cannot parse inventory for host %v", host.ID)
+			return nil, err
+		}
 		return &models.ClusterHostRequirementsDetails{
-			CPUCores: workerCPU,
-			RAMMib:   workerMemory,
+			CPUCores: WorkerCPU,
+			RAMMib:   WorkerMemory + overhead,
 		}, nil
 	}
 	return nil, fmt.Errorf("unsupported role: %s", host.Role)
 }
+
+func (o *operator) getDevicesMemoryOverhead(host *models.Host) (int64, error) {
+	if host.Inventory == "" {
+		return 0, nil
+	}
+	inventory, err := parseInventory(host.Inventory)
+	if err != nil {
+		return 0, err
+	}
+	gpuCount := o.getGPUCount(inventory)
+	return conversions.GibToMib(gpuCount), nil
```

Comment:
sounds like we are converting number of gpus to megabytes ? Maybe i miss something?

Reply:
Yeah, in this context each GPU is one GiB of additional memory. Soon it will be gpuCount+sroivNicsCount.

Reply:
Added comment.